### ISSUE TITLE
Name proposals

### ISSUE THREAD

**Issue by Sarah-Seo** [Sarah-Seo/Pad#1]
## Candidates

- DevDepo
- CodeCache
- DevCache 
- SourceNote
- CodeStash
- ~~DevStash~~ Occupied
- CodeLog
- ~~Markus~~ Occupied
- Scribe
- LastPad
- LastNotepad
- FinalPad
- Notepedia
- BlotterPad
- Inpad
- DevCodex
- ArcaneLogger

**Comment by tejado** [comment]
I'm for Inpad or LastPad :)

SourceNote is already a company, Scribe too and also already some old HTML editor. ArcaneLogger sounds like a logging solution.

**Comment by bkjohnson** [comment]
How about `Noted`?

**Comment by twhiting** [comment]
@bkjohnson seems Noted is taken as well Google "Noted company"

**Comment by bkjohnson** [comment]
I keep googling "app" instead of "company", no wonder I don't find anything :p

**Comment by tejado** [comment]
Yeah, BlotterPad was just a silly idea. I searched for english sketchbook words, found blotter and suffixed Pad :D

**Comment by Sarah-Seo** [comment]
I like both Dex and Inpad.😄
For me, Dex sounds a little better and seems to be rememberable easily.

**Comment by twhiting** [comment]
I like Dex / Noted

**Comment by tejado** [comment]
As "Noted" is a english word, I wouldn't recommend it, e.g. it will never be on the top search results. Also there is a note app in the Android store with this name. Dex too.

**Comment by bkjohnson** [comment]
I'm not too worried about the Dex Android app.  It is a Romanian dictionary that hasn't been updated since 2012.

**Comment by twhiting** [comment]
What about `Codex`?

**Comment by bkjohnson** [comment]
I thought of that too, but found that there was already a [codex.com]() and [codex.online]().  On second thought it's similar to the `Dex` situation and should still be considered.

**Comment by huettenhain** [comment]
I am totally voting for Dex.
